fastlane/lanes/localization.rb

```diff
@@ -379,4 +381,20 @@
       lint_task: 'lintWordpressVanillaRelease' # TODO: Should we adapt this?
     )
   end
+
+  # Updates the `.po` file at the given `po_path` using the content of the `sources` files, interpolating `release_version` where appropriate.
+  # Internally, this calls the `an_update_metadata_source` release toolkit action and adds Git management to it.
+  #
+  def update_po_file_for_metadata_localization(po_path:, sources:, release_version:, commit_message:)
+    ensure_git_status_clean
+
+    an_update_metadata_source(
```

Comment:
😄 

Yes, it is. This is where the removed wrapped action called this one:

https://github.com/wordpress-mobile/WordPress-Android/blob/c35747e67c270dff5c1e64cff115ac7fd4617742/fastlane/actions/android_update_metadata_source.rb#L8-L10

For reference, here's the whole of what the action did, which should be the same as what this new local method does:

https://github.com/wordpress-mobile/WordPress-Android/blob/c35747e67c270dff5c1e64cff115ac7fd4617742/fastlane/actions/android_update_metadata_source.rb#L4-L23

Reply:
> should be the same as what this new local method does:

Well, one thing I left out from the method and forgot to call out is that we no longer add and commit changes to the source files. That didn't seem right in the first place: The action is about updating the `.po` from those source files. It doesn't not touch them.

Reply:
Wait seriously so the local action was just a wrapper around calling the `release-toolkit` one, + some git operations around? And here I though it was a copy/paste of the code from `release-toolkit`… 😅 

That's actually good news then, as it makes it even easier to understand that this were doing the same (+ git), while if that were a copy/paste of the implementation that could have potentially been a trickier diff to check 😅 

Perfect then; we've already slowly migrated the trend of making our `Fastfile` now do the `git commit` and more importantly `git push` operations (rather than making the `release-toolkit` actions do it) as it felt a better separation of concern… so migrating to this here (i.e. calling `an_update_metadata_source` from the `Fastfile` and having to `git add`/`git commit` ourselves afterwards like you did in the PR) is the way to go imho 🙂

Reply:
> That didn't seem right in the first place: The action is about updating the .po from those source files. It doesn't not touch them.

Agreed — and nice catch!